packages/cache/src/internal/blob/download-cache.ts

```diff
@@ -3,13 +3,14 @@ import * as core from '@actions/core'
 import {
   BlobClient,
   BlockBlobClient,
-  BlobDownloadOptions
+  BlobDownloadOptions,
+  BlobDownloadResponseParsed
 } from '@azure/storage-blob'
 
 export async function downloadCacheFile(
   signedUploadURL: string,
   archivePath: string
-): Promise<{}> {
+): Promise<BlobDownloadResponseParsed> {
```

Comment:
looks like we don't actually use this return value:
https://github.com/actions/toolkit/blob/20ed2908f19538e9dfb66d8083f1171c0a50a87c/packages/cache/src/cache.ts#L272

wonder if it has any fields that would be useful for debug logging

Reply:
Good idea, addressed!